src/api/jest.polyfills.ts

```diff
@@ -0,0 +1,30 @@
+// jest.polyfills.js
+/**
+ * @note The block below contains polyfills for Node.js globals
+ * required for Jest to function when running JSDOM tests.
+ * These HAVE to be require's and HAVE to be in this exact
```

Comment:
does this note still apply?

Reply:
Uhmmm good question. I somehow missed it. It seems to be working without it so unsure? I'd say it doesn't?